resources/user.py

```diff
@@ -4,25 +4,41 @@
 from passlib.apps import custom_app_context as pass_context
 from models import dtlogin, dtusers
 
+from flask_jwt_extended import (create_access_token,create_refresh_token,
+jwt_required, jwt_refresh_token_required,get_jwt_identity, get_raw_jwt)
 
 auth = HTTPBasicAuth()
```

Comment:
I guess you want to use this for routes authentication but you already implemented the `flask_jwt_extended` method. I don't think you can use both implementations see the usage of [ HTTPBasicAuth()](https://github.com/miguelgrinberg/Flask-HTTPAuth)